subiquity/models/subiquity.py

```diff
@@ -105,6 +105,8 @@ def __init__(self, root, install_model_names, postinstall_model_names):
         if root != '/':
             self.target = root
 
+        self.client_variant = ''
```

Comment:
see comment on server.py. I don’t think we should store the variant here but on the application from client_variant_POST.